src/uct/sm/mm/mm_pd.h

```diff
@@ -18,16 +18,37 @@
 /* Shared memory ID */
 typedef uint64_t uct_mm_id_t;
 
+
+/*
+ * Descriptor of the mapped memory
+ */
+typedef struct uct_mm_mapped_desc {
+    void *address;    /**< local memory address */
+    uint64_t cookie;  /**< cookie for mmap, xpmem, etc. */
```

Comment:
i think 'cookie' isn't the best name here. cookie is used for knem. 
maybe we should use a more generic name that would fit all the mm components. 
mem_id or id...